artview/components/plot.py

```diff
@@ -112,23 +103,26 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, \
         self.cm_names = [m for m in cm.datad if not m.endswith("_r")]
         self.cm_names.sort()
 
-        # Create a figure for output
-        self._set_fig_ax(nrows=1, ncols=1)
-
         # Create tool dictionary
         self.tools = {}
-        self.tools['zoompan'] = None
+
+        # Check the file type and initialize limts
+        # Create a figure for output
+        self._check_file_type()
```

Comment:
I'll need to look at this, but I know the _set_fig_ax function needs _check_file_type to be run first to get the proper limits set. In turn the LaunchGUI requires the all of this to be done so that the "canvas" can be set up properly. 

That is why it has persisted in two locations. Maybe there is an elegant way to treat this problem, but I haven't figured it out yet.